src/v0/destinations/google_adwords_enhanced_conversions/transform.js

```diff
@@ -77,6 +77,8 @@ const processTrackEvent = async (metadata, message, destination) => {
   const payload = constructPayload(message, updatedMapping);
 
   payload.partialFailure = true;
+  // ref: https://developers.google.com/google-ads/api/docs/release-notes#:~:text=Added%20job_id%20as,UploadConversionAdjustmentsRequest
+  payload.jobId = metadata.jobId;
```

Comment:
@shrouti1507 There's no point of introducing `jobId` here without handling partial failure feature